src/createStatusBarItem.ts

```diff
@@ -1,13 +1,13 @@
-import { window, StatusBarAlignment } from 'vscode'
+import { window, StatusBarAlignment, StatusBarItem } from 'vscode'
 import { COMMAND_NAME } from './constants'
 import getStatusBarText from './helpers/getStatusBarText'
 
 // Taken from the `prettier-vscode` repo.
 const PRETTIER_STATUS_BAR_PRIORITY = -1
 const TOOLTIP_TEXT = 'Enable/Disable formatting'
 
-const createStatusBarItem = () => {
-  const statusBarItem = window.createStatusBarItem(
+const createStatusBarItem = (): StatusBarItem => {
+  const statusBarItem: StatusBarItem = window.createStatusBarItem(
```

Comment:
The `vscode` package is typed so TypeScript will automatically infer the types (also for `getConfiguration`), it shouldn’t be necessary to be explicit here, nor in the return type?

Reply:
I think the same applies here — bad VSCode setup on my part throwing false-positives.